patterns/2-structured/document-your-guiding-principles.md

```diff
@@ -124,6 +124,7 @@ All Trusted Committers of a project are published.
 * Europace AG
 * GitHub
 * Robert Bosch GmbH
+* Siemens' InnerSource strategy is built upon the guiding principles of Documentation, Workflow, Transparency, Continuous Integration and Culture which are detailed in their InnerSource portal.
```

Comment:
@fgreinacher if you have more information about these guiding principles that you can share publicly, then you can also break this out into a dedicated section below. See the examples from Europace, GitHub, Bosch below.

Not a necessity though, just a nice to have if it is quick and easy :)

Reply:
@spier That would need some amount of cleanup I currently don't have the time for, sorry. Also nothing too surprising in there I'd say :) Let's leave it like that for now.